app/models/weam.rb

```diff
@@ -82,8 +90,8 @@ class Weam < ImportableRecord
 
   def institution_type
     msg = 'Unable to determine institution type'
-    errors.add(:institution_type, msg) unless ['OJT', 'PRIVATE', 'FOREIGN', 'CORRESPONDENCE',
```

Comment:
Are the quotes supposed to be here? Seems like you want the constant (which evaluates to what you have in quotes)

Reply:
I switched it to an array of constants representing each of the previous values.